README.md

```diff
@@ -44,17 +40,18 @@ If the svg sprite is included at the top of every HTML document in your React pr
 0. Check out a new branch. For example, `new_fancy_icon`.
```

Comment:
Some minor tweaks suggested for the "Adding new icons" section:
```
 #### Adding new icons
0. Check out a new branch. For example, `add_fancy_icon`
1. Use Sketch and Adobe Illustrator template in `design_resources/` folder to design your new icon.
  - Adobe Illustrator template, `IconGrid.ai`, contains the icon grid provided by Victor
  - Sketch template, `IconExport.sketch`, contains the basic setup for an icon Sketch file
2. Save your icon's Sketch file to `src/sketch/`
3. Run `yarn run build` to export icon distributions from Sketch files
4. Commit the resulting changes (more info below)
5. Submit a pull request (more info below)
6. The PR will publish a `-beta` tag to npm
```

Since designers don't use Git too often, might be good to add a brief how-to for using Git:
```
#### Committing changes to Github
0. `cd ~/Documents/swarm-icons/` to change to the icons repo
1. `git checkout -b add_iconName_icon` to check out a new branch
2. `git add --all && git commit -m "your comment"`to add new files, commit new files (put them in the shipping box) and make a comment (in the quotes after `-m`) that explains what you're adding or changing
3. `git pull --no-edit origin` to retrieve any new changes from Github
4. `git push origin` to send this batch of new files and/or changes to Github
5. Come back to [https://github.com/meetup/swarm-icons/](https://github.com/meetup/swarm-icons/) to submit a pull request

If you're really uncomfortable with the command line, download the [Github Desktop app](https://desktop.github.com/)
```

Should we note some naming guidelines somewhere?
```
#### File management guidelines
* Each platform has it's own Page in Sketch. This helps us manage platform splits more easily.
* Name Artboards using dashes, not camelCase. For example: `arrow-right`, not `arrowRight`
* Create a new Artboard for any variant of an icon. Variants get named using a double dash after the base icon name. For example: `arrow-right--small`, `arrow-right--pressed`
```